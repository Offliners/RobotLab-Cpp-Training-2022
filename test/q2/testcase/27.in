30
BesIdes
QuiTE
wHeNeVeR
patient
bend
bIOlOgY
rUbbER
tRaiN
lOgiC
PrECISE
BADMINTOn
lifEBoat
microWave
lorry
pErIOdiC
HERaLD
portray
cAPItaL
MIDniGhT
pHIlOSophIcaL
Can
MODeRNIZE
pOliTicAL
AuTUmN
mUtton
waste
gUARaNtEE
conceDe
CONNECT
gOOSe
