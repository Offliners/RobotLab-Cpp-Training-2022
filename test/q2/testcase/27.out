prep./adv. 除此之外
adv. 完全；非常
adv./conj. 無論什麼時候；每當
adj. 有耐心的
n./v. 彎曲
n. 生物學
n. 橡膠
n. 火車
n. 邏輯
adj. 精確的
n. 羽毛球
n. 救生艇
n. 微波
n. 載貨卡車
adj. 定期的
n./v.使者；宣布 v. 之將臨
v. 描繪
adj./n. 主要的；首都；資本
n. 午夜
adj. 哲學的
n./v. 罐頭；金屬容器
v. 使現代化
adj. 政府的
n. 秋季
n. 羊肉
n./v. 浪費
n./v. 保證；擔保品
v. 讓與
v. 連結
n. 鵝；傻瓜
